index.js

```diff
@@ -693,6 +718,11 @@ var tick = function() {
 					delete node.transformBegin;
 					delete node.transformTo;
 				}
+				// check if its time to unfreeze the node
+				if (node.frozen != 0 && time - node.frozen >= GAMERULES.FREEZE_TIME) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2569284612/issues/source?bid=3248399&fileBranchId=3297449#l722)